### DISCUSSION TITLE
A "stop text wrapping" within page method?

### DISCUSSION THREAD

**Discussion by maphew** [TriliumNext/Trilium#6019] [Ideas]
After an image or other block element after it's been floated left or right, is there a way to stop wrapping text and other page elements around it?

In this screenshot, I want 3 sections and within each sections to have one image floated right with text wrapping left around it. For the 1st section it works, but then the following sections are sucked into the 1st, also wrapping. The read lines indicate a section break, which should be flat horizontal. 2nd screenshot shows desired look, but it's been achieved by inserting blank paragraphs, but that's not reliable (resizing the window changes the flow).

![image](https://github.com/user-attachments/assets/1b0a3051-6432-4a03-ae3d-c94406070910)

![image](https://github.com/user-attachments/assets/955ff425-9099-4dd4-850f-192aacbbd8de)

**Comment by eliandoran** [comment]
@maphew , before this can be implemented we also need to define exactly how the "clear" (CSS terminology for shifting elements underneath floating elements such as images) is inserted by the user. And also how it can be removed.

Most intuitive one would be similar to how we have support for page breaks, but they would look kind of ugly in this context. There is also the possibility of modifying horizontal separators (`<hr/>`) to have this feature.

**Comment by maphew** [comment]
Adding to `<hr/>` makes the most sense to me. A mark in it's favour is that it's visual, which would help with knowing it's there, and with removal. 

Adding clear as a property to something else, such as the next image, can be problematic because outside of observing the behaviour you don't know it's there until inspecting properties. This is how I've seen other programs do it though.

**Comment by CobriMediaJulien** [comment]
Could we just add to the allrdy exisiting horizontal line a "clear" ?

![image](https://github.com/user-attachments/assets/b276ba4e-cb8c-48f6-9e25-a1ad6cfb7bd0)
